snaps/features/custom-ui/index.md

```diff
@@ -1064,6 +1246,8 @@ Outputs text.
 - `alignment` - (Optional) The alignment of the text.
   Possible values are `"start"`, `"center"`, and `"end"`.
   The default is `"start"`.
+- `children` - The content to display.
+  This can include strings, [`Bold`](#bold), [`Italic`](#italic), [`Icon`](#icon), [`Link`](#link), and [`Skeleton`](#skeleton) components.
```

Comment:
```suggestion
  This can include strings, and [`Bold`](#bold), [`Italic`](#italic), [`Icon`](#icon), [`Link`](#link), and [`Skeleton`](#skeleton) components.
```